build/Targets/VSL.Settings.targets

```diff
@@ -136,6 +136,11 @@
     <HighEntropyVA>true</HighEntropyVA>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Host processes are only really useful for a few of our programs; most need command line arguments -->
```

Comment:
This comment confuses me. The hosting process and needing command line arguments are two orthogonal things. I'm not saying the change is wrong, there's just a long history of this file getting things added for poorly commented reasons and then us scratching our head years later wondering why. :smile: